VimeoUpload/UIKit/Model/VIMALAsset.swift

```diff
@@ -40,7 +40,7 @@ import AVFoundation
     {
         get
         {
-            return self.alAsset.defaultRepresentation().url().absoluteString
```

Comment:
I deleted this class is the master branch, probably worth merging master up into v1.0. I should have done this